src/commands/refocus-collector-stop.js

```diff
@@ -13,20 +13,32 @@
  */
 const program = require('commander');
 const logger = require('winston');
-const cmdStart = require('./stop');
-const debug = require('debug')('refocus-collector:commands');
+const cmdUtils = require('./utils');
+const doPost = require('../utils/httpUtils.js').doPostToRefocus;
 
 program
-  .option('-f, --force', 'Stop the collector without flushing the samples')
+  .option('-n, --collectorName <collectorName>', 'The name of the ' +
+    'collector to be started')
+  .option('-u, --refocusUrl <refocusUrl>', 'The url of the refocus ' +
+    'instance this collector will send to')
+  .option('-t, --accessToken <accessToken>', 'A valid refocus token')
```

Comment:
What you said is one reason. Also, when you are running any collector command, each is starting its own process and there is no way to access the memory of the other process.

Reply:
Makes sense. Doesn't seem very user friendly though for the case where you want to stop/pause/resume the collector from the same place it's running. As a user, I would expect to just be able to run "refocus-collector stop".

Reply:
@jgraff2 I like that model better, too, but it will require a whole other layer of process manager infrastructure on the collector side, which we should certainly consider for *v.next*